app/models/region.rb

```diff
@@ -27,6 +27,30 @@ def protected_areas_with_iucn_categories
     )
   end
 
+  def protected_areas_per_governance
+    region_data = Hash.new { |hash, key| hash[key] = Hash.new }
+    processed_data = []
+
+    countries.each do |country|
+      country.protected_areas_per_governance.each do |protected_area|
+        region_pa_category = region_data[protected_area["governance_name"]]
+        region_pa_category["governance_type"] = protected_area["governance_type"] if region_pa_category["governance_type"].nil?
+        region_pa_category["count"] = [] if region_pa_category["count"].nil?
+        region_pa_category["count"] << protected_area["count"].to_i
```

Comment:
You can also probably improve this and the percentage by doing: `region_pa_category["country"] += protected_area["count"].to_i` meaning that you are already summing up the values here; so you don't need to do the `reduce` afterwards

Reply:
Thanks @Levia, should I also do this for the `region_pa_category["percentage"]` ?

Reply:
Yep, it should work the same way. Just remember to divide by the count where you are already doing that